src/app/core/store/navigation/query.util.ts

```diff
@@ -106,3 +107,7 @@ export function conditionFromString(condition: string): ConditionType {
   }
   return null;
 }
+
+export function queryIsNotEmpty(query: QueryModel) {
+  return query && Object.values(query).find(val => val instanceof Array ? val.length > 0 : val);
```

Comment:
I think we should always specify the return parameter of a function unless it is `void`.